python/pynq/mmio.py

```diff
@@ -96,16 +97,18 @@ def __init__(self, base_addr, length = 4, debug = False):
         self.length = length
```

Comment:
For line 92, it seems that `stop` is no longer used, can we remove this variable completely?